inst/application/GenericQC/QC_ProteinAbundances.Rmd

```diff
@@ -88,12 +118,14 @@ table <- DT::datatable(
 
 The columns of the table below contains:
 
-- protein_Id - protein identifier
-- nrPeptides - nr of peptides per protein
-- description - protein description from fasta header
-- nrMeasured_<GroupName> - How often a protein was observed in each of the groups and overall
-- meanAbundance_<GroupName> - average protein abundance (see 1. at the bottom of the document) per group.
-- signal_percent_<GroupName> - percent of the total signal of all identified proteins measured by the mass spectrometric instrument attributed to each protein.
+- `r strings[["feature_id"]]` - `r strings[["target"]]` identifier
+`r cat(ifelse(params$target_type == "metabolite", "", "- nrPeptides \n- nr of peptides per protein"))`
+- description - `r strings[["target:description"]]`
+- nrMeasured_<GroupName> - How often a `r strings[["target"]]` was observed in each of the groups and overall
+- meanAbundance_<GroupName> - average `r strings[["target"]]` abundance (see 1. at the bottom of the document) per group.
+
```

Comment:
`- signal_percent_<GroupName> - percent of the total signal of all identified proteins measured by the mass spectrometric instrument attributed to each protein.`

This was deleted. Because I did not see it in the existing workunits anymore, but please correct me if I am wrong.